constructor/main.py

```diff
@@ -178,10 +172,9 @@ def main_build(
             if isinstance(path, str):
                 new_extras.append(abspath(join(dir_path, path)))
             elif isinstance(path, dict):
-                assert len(path) == 1
-                orig, dest = next(iter(path.items()))
-                orig = abspath(join(dir_path, orig))
-                new_extras.append({orig: dest})
+                for orig, dest in path.items():
+                    orig = abspath(join(dir_path, orig))
+                    new_extras.append({orig: dest})
```

Comment:
We now allow multiple entries in the same dict (why not?), mostly because I couldn't figure how to limit the length of an arbitrary-keys dictionary with JSON Schema. If you feel this restriction is important we can add the runtime check back.